src/libs/util/include/storm/fs.h

```diff
@@ -11,7 +11,7 @@ using namespace std::filesystem;
 
 inline path GetStashPath()
 {
-    static path path;
+    path path;
```

Comment:
`SHGetKnownFolderPath`/`CoTaskMemFree` are relatively heavy
instead, we can e.g. add a default boolean arg which will turn it to work with a stack variable if set.
OR to keep the path in a static buffer (but size is a problem in that case, could be up to 2^16 for Unicode)
OR even to inline the code manually into `mimalloc_fun` because we need such only there so far
Anyway, at some point I would like to rework all that crappy `fs.h` stuff